### PR TITLE
Markdown update and CI

### PR DESCRIPTION
**Pull request by asklar**
This PR adds a CI loop to use [unbroken](https://github.com/asklar/unbroken) to verify that there are no broken links in markdown.
It also fixes a few broken links that were dangling where files they point to had gotten deleted/moved around. 
Addresses #4191 #4339 

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/react-native-windows/pull/4409)

### CODE REVIEW

**Comment by NickGerleman** [comment]
This is nifty! A couple questions though:

1. What's up with all the new files? Were docs moved around?
2. Is this a required check? It could get annoying if a link breaking externally would block checkin of unrelated changes.

**Comment by NickGerleman** [comment]
It also seems like this might be running twice per PR?

**Comment by asklar** [comment]
> This is nifty! A couple questions though:
> 
> 1. What's up with all the new files? Were docs moved around?
> 2. Is this a required check? It could get annoying if a link breaking externally would block checkin of unrelated changes.

1. the files got moved around a couple of times, they ended up under \current\docs, and that folder got deleted in PR #4272 
2. This doesn't currently block. We should figure out a way to surface the status but at least this is a first step towards ensuring we don't have broken links

**Comment by asklar** [comment]
could you clarify what are you seeing this run twice? :)

---
In reply to: [602957617](https://github.com/microsoft/react-native-windows/pull/4409#issuecomment-602957617) [](ancestors = 602957617)

**Comment by NickGerleman** [comment]
> could you clarify what are you seeing this run twice? :)
> 
> In reply to: [602957617](https://github.com/microsoft/react-native-windows/pull/4409#issuecomment-602957617) [](ancestors = 602957617)

For listed checks of this PR we see both "push" and "pull request", taking roughly the same time each. It seems like they might be doing the same thing.

**Comment by kikisaints** [comment]
@asklar is this PR putting the getting started md back into our main repo? If so, I ask that you make this PR [over here](https://github.com/microsoft/react-native-windows-samples/blob/master/docs/getting-started.md). 

As from now all all guidance and instructions **not** related to the main repo directly, should go there.

File: docs/GettingStarted.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+- Run `react-native start` from your project directory, and wait for the React Native packager to report success.
+- Click the `Run` button to the right of the platform combo box control, or select the `Debug`->`Start without Debugging` menu item. You now see your new app! :tada:
+
+*Note:* You should **only** modify the project and source files for your app (e.g. `AwesomeProject`). The files for the "ReactNative" and other projects shown in the Visual Studio solution are in the `node_modules` directory (which will not be committed to your source repository since it is ignored in `.gitignore`). Any changes to files in `node_modules` will be overwritten when doing an `npm install` or `npm update`. If you need to add a new native module or override some React Native behavior, see [Extending React Native](http://github.com/microsoft/react-native-windows#extending-react-native)
+
+#### Without Visual Studio
+
+In your React Native Windows project directory, run:
+```
+react-native run-windows
```
Comment (rectified95) [inline docs/GettingStarted.md]: Should this be 'npx react-native run-windows'? In another issue, I'm updating the CLI prompt to say that, so thought it should be the same. Also, currently https://microsoft.github.io/react-native-windows/docs/getting-started says 'npx..' as well.

Comment (asklar) [reply docs/GettingStarted.md]: totally possible (read, likely) that we need to revise the docs - let's do a pass at them once they're checked in and CI running

**Comment by asklar** [comment]
> @asklar is this PR putting the getting started md back into our main repo? If so, I ask that you make this PR [over here](https://github.com/microsoft/react-native-windows-samples/blob/master/docs/getting-started.md).
> 
> As from now all all guidance and instructions **not** related to the main repo directly, should go there.

renamed this GettingStarted to GettingStarted-Contributors. We'll need to do a pass at it and update or may end up removing it altogether.

**Pull request closed without merging**